README.md

```diff
@@ -32,6 +32,8 @@
 * `pip install dbt-athena-community`
 * Or `pip install git+https://github.com/dbt-athena/dbt-athena.git`
 
+_**Note:** in order to avoid potential conflicts, make sure `dbt-athena-adapter` is not installed in the target environment._
```

Comment:
@nicor88 I am not really sure what this should so I'd be happy to take any suggestions here 🙂

Reply:
How about adding a note in https://github.com/dbt-athena/dbt-athena#known-issues ??

Reply:
@nicor88 sure, let's try that!